native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/ListenerUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.graphql.runtime.engine;
+
+import io.ballerina.runtime.api.values.BObject;
+
+/**
+ * External utility methods used in Ballerina GraphQL listener.
+ */
+public class ListenerUtils {
```

Comment:
Since, this is a utility class we could mark it as `final` (since we do not need developers to create sub-classes out of this).
```suggestion
public final class ListenerUtils {
```